sdv/model_generator/main.py

```diff
@@ -88,12 +89,12 @@ def main():
         default="",
         help="Whitelisted extended attributes as comma separated list. Note, that "
         "extended attributes aren't considered by the generator. This paramter is "
-        "only for suppressing warnings/errors."
+        "only for suppressing warnings/errors.",
     )
     parser.add_argument(
-        "vspec_file",
-        metavar="<vspec_file>",
-        help="The vehicle specification file to convert.",
+        "import_file_path",
```

Comment:
`import_file_path` sounds like we are really importing something into a running system.
I would recommend we name this `input_file_path`